cdserver/app/models/shelter_building.rb

```diff
@@ -2,10 +2,19 @@
 class ShelterBuilding < ApplicationRecord
   has_many :floors
   has_many :units, through: :floors
-  has_one :address
+  belongs_to :address
```

Comment:
not sure if belongs_to is the right thing.  What happens if the building were to be moved?
What I mean is-- the "Mens Shelter BLG 1" is no longer referred to by 10 west 42nd street, it is now referred to as 1110 avenue of the americas
That happens all the time.  the shelter cannot be destroyed if the address gets changed.